lib/installedtest.pm

```diff
@@ -50,6 +50,7 @@ sub exit_user_console {
     # terminal TTY for the next user_console() call is not a no-op (and hence
     # causes the display to refresh).
     console_user_exit();
+    sleep 4;
```

Comment:
Why is this blocking delay needed before calling `send_key`? There is another one right after calling `send_key`. If it's needed, please add a comment with an explanation. Other delays in this file have a comment like "There's a timing problem when...".

Reply:
Is this the sporadic failure you were getting? If so, can the other `sleep` be increased from 4 to 8? Or has to be another `sleep` right before the `send_key`?